lib/matplotlib/axes/_base.py

```diff
@@ -1287,25 +1287,59 @@ def apply_aspect(self, position=None):
 
     def axis(self, *v, **kwargs):
         """
-        Convenience method for manipulating the x and y view limits
-        and the aspect ratio of the plot. For details, see
-        :func:`~matplotlib.pyplot.axis`.
+        Call signature::
+
```

Comment:
Not quite, I think you will need to escape those *s with backslashes, and also I don't think you need :meth: or :func:, though I may have got that wrong.

Before you commit you can check it works by switching to the ``matplotlib/doc`` folder and running ``make.py html`` which will generate the doc and place it in ``doc/build/html``.